README.md

```diff
@@ -87,6 +87,23 @@ Show message with ID 1
 
     RETR 1
 
+## Use with docker-machine
+
+Because service clients, like the addon-administration, that are using the
+3-legged flow to authenticate a user with a redirect to OSIAM only know one
+endpoint to redirect to and back to themselves. This container will only
```

Comment:
Yes, the sentence made no sense. I removed it, because this section should only explain how to run the 3-legged flow with the addon-administration when using docker-machine